### DISCUSSION TITLE
how  ant Table is automatically rerendered when datasource data changes ib blazor

### DISCUSSION THREAD

**Discussion by Hunglmc** [ant-design-blazor/ant-design-blazor#3684] [Q&A]
Happy new  year!!. I have a question, i have modal with code  this and  how Table is automatically rerendered when datasource data changes ib blazor `  private async Task<ModalRef> Delete(Guid id, string EmplNo = "")
  {}`

**Comment by ElderJames** [comment]
Thanks for contacting us @Hunglmc, if your DataSource was changed on the page, the tablel would rerender automatically. If your datasource is update on server, you need to reload it first.
